oracle/packages/node/.env.example

```diff
@@ -0,0 +1,4 @@
+API_URL="https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=USD&api_key=<your api key here>"
+ORACLE_CONTRACT_ID=0x1bc1332bff931c3cfc9752fe10fe30e1ed3c41f77101f7d144c8183355e5590e
+WALLET_SECRET=0x0000000000000000000000000000000000000000000000000000000000000001
```

Comment:
Does this secret key matter; can it be set to anything; must it be a real secret key from `forc-wallet`?